src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -1041,7 +1078,8 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 			Analytics.collect(.keySubmissionMetadata(.lastSubmissionFlowScreen(.submissionFlowScreenSymptomOnset, coronaTestType)))
 		}
 
-		let exposureSubmissionSymptomsOnsetViewController = ExposureSubmissionSymptomsOnsetViewController(
+		var exposureSubmissionSymptomsOnsetViewController: ExposureSubmissionSymptomsOnsetViewController!
```

Comment:
Same

Reply:
The controller pointer cannot be nil, because he will instantiated in the next line. It's like:

```swift
var result: String!
switch some {
case .foo: result = "Hello"
case .bar: result = "World"
}

// result cannot be nil
```

As discussed in the chat, we'll keep that here, cause of something like this

```swift
let topBottomContainerViewController = TopBottomContainerViewController(
	topController: exposureSubmissionSymptomsViewController ?? UIViewController(),
	bottomController: footerViewController
)
```
is not so elegant with `?? UIViewController()`